test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetSpecCommandTest.cs

```diff
@@ -56,6 +56,9 @@ public void SpecCommand_NoProjectFile()
       </group>
     </dependencies>
   </metadata>
+  <files>
+    <file src=""$icon_file_name_within_the_package_here_or_delete_this_line$"" target="""" />
```

Comment:
Shouldn't the file element also be commented out? Doesn't pack fail when there's a file specified that can't be found?

Reply:
I tested, it still works, see output in desc part. 
I thought make life bit easier for someone using it since it still works.

Reply:
out of curiosity why does it fail?
I'm guessing the replacement token has no value and somewhere in the pack code `""` is excluded?

Reply:
Sorry, it was my mistake. I was testing on existing repo. But it fails on empty repo. 
I addressed it.